tests/test_models/test_engine/test_db_storage.py

```diff
@@ -68,8 +68,61 @@ def test_dbs_func_docstrings(self):
                             "{:s} method needs a docstring".format(func[0]))
 
 
-class TestFileStorage(unittest.TestCase):
-    """Test the FileStorage class"""
+class TestDBStorage(unittest.TestCase):
```

Comment:
Consider testing that the `dbstorage` object has all of the required methods. Could try checking that `hasattr(self.dbstorage, 'new')` is `True` for example.